backend/tests/test_indexing.py

```
@@ -924,5 +924,288 @@ def bad_callback(current, total, message=None):
+
+        with patch('backend.indexing.get_tags', return_value="test"), \
+             patch('backend.indexing.perform_global_clustering', return_value={0: [0]}), \
+             patch('backend.indexing.smart_summary', return_value="Summary"):
+
+            res = create_index(
+                self.temp_dir,
+                provider="openai",
+                api_key="fake_key",
+                embedding_client=mock_embedding_client
+            )
+            index, docs, tags, idx_sum, clus_sum, clus_map, bm25 = res
```

Ruff (0.15.5)

[warning] Unpacked variable `docs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)